src/main/java/net/minecraftforge/event/TickEvent.java

```diff
@@ -60,10 +78,20 @@ public ClientTickEvent(Phase phase)
 
     public static class WorldTickEvent extends TickEvent {
         public final Level world;
-        public WorldTickEvent(LogicalSide side, Phase phase, Level world)
+        private final BooleanSupplier haveTime;
+        public WorldTickEvent(LogicalSide side, Phase phase, Level world, BooleanSupplier haveTime)
         {
             super(Type.WORLD, side, phase);
             this.world = world;
+            this.haveTime = haveTime;
+        }
+
+        /**
+         * See {@link ServerTickEvent#haveTime()}.
+         */
```

Comment:
This should copy the javadoc from that method and use a `@see` tag to link the two.